apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/nlog-template.xml

```diff
@@ -7,8 +7,8 @@
   </extensions>
   
   <targets async="true">
-    <target name="stackdriver" xsi:type="GoogleStackdriver" projectId="PROJECT_ID" logId="LOG_ID">
-      <!-- Potentially additional configuration -->
+    <target name="stackdriver" xsi:type="GoogleStackdriver" projectId="PROJECT_ID" logId="LOG_ID" includeEventProperties="true" sendJsonPayload="true">
```

Comment:
The intention is for this to be a pretty minimal example - I'd rather not include those optional configuration values in there. (While the log ID and project ID are optional in some cases, they're often the difference between "working" and "not working at all" whereas the format of the log entry is a bit different.)

I wouldn't mind having an extra section in here talking about the pros and cons of using a JSON payload, but I don't want it right at the start of the docs.

Reply:
I have now reverted the change of `nlog-template.xml` and instead introduced `nlog-jsonPayload.xml`.